### DISCUSSION TITLE
What is the point of fabric.Point ?

### DISCUSSION THREAD

**Discussion by rahgurung** [fabricjs/fabric.js#8635] [Q&A] [answered]
As far as I understood it is nothing but x and y coordinates with methods available to do a bunch of stuff like setting them, finding mid points etc. We cannot draw it on canvas, what should be the ideal scenario to use it?

**Comment by ShaMan123** [comment]
It is used across fabric for calculations and thus the methods it exposes
Look around in the source code

**Comment by asturur** [comment, marked as answer]
Good to know is confusing, maybe is something we can clarify with docs and also provide an example on how to use points in an app ( for example collecting data clicked with the mouse, or somethin like that )
most of the time you won't create points, but you will interact with Points created by fabric code
